Chinese/dream/dream.md

```diff
@@ -0,0 +1,7 @@
+你來到這就是在作夢~ 來吧! 以下自由選擇:
+
+1. [我要穿越](https://github.com/xbane123/create-your-own-adventure/blob/master/Chinese/Yifan/modernCity.md)
+
+2. [我要當至尊寶](https://github.com/xbane123/create-your-own-adventure/blob/master/Chinese/lotus/%E5%90%83%E9%A5%BC%E5%B9%B2%E9%81%90%E6%83%B3.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.